arbitrum-docs/stylus/stylus-quickstart.md

```diff
@@ -11,209 +11,242 @@ import PublicPreviewBannerPartial from '../partials/_public-preview-banner-parti
+  verify      Verify the deployment of a Stylus contract [aliases: v]
+  cgen        Generate c code bindings for a Stylus contract
+  replay      Replay a transaction in gdb [aliases: r]
+  trace       Trace a transaction [aliases: t]
+  help        Print this message or the help of the given command(s)
 
 Options:
   -h, --help     Print help
   -V, --version  Print version
 ```
 
-### Overview
-
-The cargo stylus command comes with useful commands such as `new`, `check` and `deploy`, and `export-abi` for developing and deploying Stylus programs to Arbitrum chains. Here's a common workflow:
+### Creating a project
 
-Start a new Stylus project with
+You can create a new Stylus project by running:
 
-```bash
+```shell
 cargo stylus new <YOUR_PROJECT_NAME>
 ```
 
-The command above clones a local copy of the [stylus-hello-world](https://github.com/OffchainLabs/stylus-hello-world) starter project, which implements a Rust version of the Solidity `Counter` smart contract example. See the [README](https://github.com/OffchainLabs/stylus-hello-world/blob/main/README.md) of stylus-hello-world for more details. Alternatively, you can use `cargo stylus new --minimal <YOUR_PROJECT_NAME>` to create a more barebones example with a Stylus entrypoint locally, useful for projects that don’t need all the Solidity plumbing. Note that the term "minimal" refers to a contract that handles simple bytes in, bytes out operations. Hence, the `cargo stylus export-abi` command (introduced later) won't work with contracts that use the entrypoint style because they don't enforce the Solidity ABI.
+`cargo stylus new` generates a starter template that implements a Rust version of the [Solidity `Counter` smart contract example](https://github.com/OffchainLabs/counter_contract/blob/master/contracts/Counter.sol). 
 
-Then, develop your Rust program normally and take advantage of all the features the [stylus-sdk](https://github.com/OffchainLabs/stylus-sdk-rs) has to offer.
+Alternatively, you can create a more barebone example with a Stylus entry point locally by running `cargo stylus new --minimal <YOUR_PROJECT_NAME>`. 
+Note that the `--minimal` option refers to a contract that handles simple "bytes in, bytes out" operations. Hence, the `cargo stylus export-abi` command (introduced later) won't work with contracts that use the entry point style because they don't enforce the Solidity ABI.
 
-### Checking your Stylus project is valid
+At this point, you can move on to the next step of this guide or develop your Rust smart contract. Feel free to use the [Stylus Rust SDK reference](./reference/overview) as a starting point.
 
-To check whether or not your program will successfully **deploy and activate** onchain, use the `cargo stylus check` subcommand:
+## Checking if your Stylus project is valid
 
-```
+By running the ' cargo stylus check ' command, you can check if your program can be successfully **deployed and activated** onchain.
```

Comment:
```suggestion
By running the `cargo stylus check` command, you can check if your program can be successfully **deployed and activated** on-chain.
```

I believe the convention was to use "on-chain" instead of "onchain", but I can't find it in the Pattern guide. Adjust if necessary.